ERR\_DATASET\_INVALID\_CONFIG: Invalid dataset configuration[¶](#err-dataset-invalid-config-invalid-dataset-configuration "Permalink to this heading")
======================================================================================================================================================


The configuration of the current dataset seems invalid.



Remediation[¶](#remediation "Permalink to this heading")
--------------------------------------------------------


The first step for solving this issue is to go to the settings screen for the affected dataset. For most datasets types, you can test the settings from there.


* Refer to the [documentation on Connecting to data](../../connecting/index.html) in order to get additional information for your dataset type
* Make sure that the dataset’s settings are correct